test/functional/tools/identifier_conditional.xml

```diff
@@ -0,0 +1,15 @@
+<tool id="identifier_conditional" name="identifier_conditional">
```

Comment:
Should this id be `identifier_collection`? Seems more proper and it's what is used at https://github.com/galaxyproject/galaxy/pull/2570/files#diff-46fef38dde830e533e91e849d89bceabR809

If yes, also this file should be renamed and the change propagated to `test/functional/tools/samples_tool_conf.xml`